### PR TITLE
Replaced var keywords with let or const - / src / edit / toolbars /DistortableImage.PopupBar.js #1086

### PR DESCRIPTION
**Pull request by Achele**
Fixes #1086 (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `grunt test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updates
* [ ] @mention the original creator of the issue in a comment below for help or for a review

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by Achele** [comment]
Please review @TildaDares.

**Review by TildaDares — APPROVED** [review]
Amazing work @Achele. Thank you!

File: src/edit/toolbars/DistortableImage.PopupBar.js [hunk 1]
```diff
@@ -37,11 +37,11 @@ L.DistortableImage.PopupBar = L.Toolbar2.Popup.extend({
   },
 });
 
-L.distortableImage.popupBar = function(latlng, options) {
+L.distortableImage.popupBar = function (latlng, options) {
   return new L.DistortableImage.PopupBar(latlng, options);
 };
 
-L.DistortableImageOverlay.addInitHook(function() {
+L.DistortableImageOverlay.addInitHook(function () {
```
Comment (TildaDares) [inline src/edit/toolbars/DistortableImage.PopupBar.js:44]:
```suggestion
L.DistortableImageOverlay.addInitHook(function() {
```

File: src/edit/toolbars/DistortableImage.PopupBar.js [hunk 2]
```diff
@@ -37,11 +37,11 @@ L.DistortableImage.PopupBar = L.Toolbar2.Popup.extend({
   },
 });
 
-L.distortableImage.popupBar = function(latlng, options) {
+L.distortableImage.popupBar = function (latlng, options) {
```
Comment (TildaDares) [inline src/edit/toolbars/DistortableImage.PopupBar.js:40]:
```suggestion
L.distortableImage.popupBar = function(latlng, options) {
```

File: src/edit/toolbars/DistortableImage.PopupBar.js [hunk 3]
```diff
@@ -66,7 +66,7 @@ L.DistortableImageOverlay.addInitHook(function() {
     lock: L.LockAction,
   };
 
-  var a = this.options.actions ? this.options.actions : this.ACTIONS;
+  const a = this.options.actions ? this.options.actions : this.ACTIONS;
 
-  this.editing = L.distortableImage.edit(this, {actions: a});
+  this.editing = L.distortableImage.edit(this, { actions: a });
```
Comment (TildaDares) [inline src/edit/toolbars/DistortableImage.PopupBar.js:71]:
```suggestion
  this.editing = L.distortableImage.edit(this, {actions: a});
```

**Comment by TildaDares** [comment]
Congratulations on merging your PR @Achele 🎉🎉

**Comment by Achele** [comment]
Thank you @TildaDares. 
I would love to help create FTO issues. Is there a guide, or I can go through the codebase and just create one? I would really love a response. Thank you

**Comment by TildaDares** [comment]
@Achele You can take a look at other FTOs https://github.com/publiclab/Leaflet.DistortableImage/issues?q=is%3Aissue+is%3Aopen+label%3Afirst-timers-only to know how to create one. Thank you!

**Pull request merged**